tests/Api/TestCase.php

```diff
@@ -152,12 +152,16 @@ protected function head($uri): ResponseInterface {
     return $this->app->run(true);
   }
 
-  protected function assertContentType(ResponseInterface $response, $contentType): void {
+  protected function assertContentType($contentType, ResponseInterface $response): void {
     $headers = $response->getHeaders();
 
     Assert::assertTrue(
         in_array($contentType, $headers['Content-Type']),
         sprintf('Content-Type does not include "%s"', $contentType)
     );
   }
+
+  protected function assertStatus(int $code, ResponseInterface $response): void {
```

Comment:
`assertStatus` is rather generic ... `assertResponseStatus` and reverse the signature so the response is first then the status would be better...

you don't have to change it, just my opinion

Reply:
like the content type one, I ended up with the $response as the second parameter because I was thinking of functions like in_array, contains, etc... usually go (needle, haystack)